met-web/src/routes/AuthenticatedRoutes.tsx

```diff
@@ -8,17 +8,18 @@ import UserService from '../services/UserServices';
 import View from '../components/Form/View';
 import CreateEngagementForm from '../components/engagement/CreateEngagementForm';
 import Engagement from '../components/engagement';
-import Drawer from '../components/layout/Drawer/CustomDrawer';
+import SideNav from '../components/layout/SideNav/SideNav';
 
 const AuthenticatedRoutes = () => {
     let adminRole = UserService.hasAdminRole();
-
+    let width = screen.width;
     //ToDO: remove when roles are defined in keycloak
     adminRole = true;
 
     return (
         <ThemeProvider theme={BaseTheme}>
-            <Drawer />
+            {width > 1350 ? <SideNav /> : <></>}
```

Comment:
I noticed in the sideNav there was a conditional to view it according the the width > 1350 condition. But here we aren't viewing it at all under 1350. Do we want to hide under 1350 for now?